DSCResources/MSFT_xWebsite/MSFT_xWebsite.psm1

```diff
@@ -729,47 +734,47 @@ function Update-WebsiteBinding
         $HostHeader = $binding.CimInstanceProperties['HostName'].Value
         $CertificateThumbprint = $binding.CimInstanceProperties['CertificateThumbprint'].Value
         $CertificateStoreName = $binding.CimInstanceProperties['CertificateStoreName'].Value
-        $SSLFlags = $binding.CimInstanceProperties['SSLFlags'].Value
+        $SslFlags = $binding.CimInstanceProperties['SslFlags'].Value
 
-        $bindingParams = @{}
-        $bindingParams.Add('-Name', $Name)
-        $bindingParams.Add('-Port', $Port)
+        $BindingParams = @{}
+        $BindingParams.Add('Name', $Name)
+        $BindingParams.Add('Port', $Port)
 
         #Set IP Address parameter
-        if($IPAddress -ne $null)
+        if ($IPAddress -ne $null)
         {
-            $bindingParams.Add('-IPAddress', $IPAddress)
+            $BindingParams.Add('IPAddress', $IPAddress)
         }
         else # Default to any/all IP Addresses
         {
-            $bindingParams.Add('-IPAddress', '*')
+            $BindingParams.Add('IPAddress', '*')
         }
 
         #Set protocol parameter
-        if($Protocol -ne $null)
+        if ($Protocol -ne $null)
         {
-            $bindingParams.Add('-Protocol', $Protocol)
+            $BindingParams.Add('Protocol', $Protocol)
         }
-        else #Default to Http
+        else #Default to HTTP
         {
-            $bindingParams.Add('-Protocol', 'http')
+            $BindingParams.Add('Protocol', 'http')
         }
 
         #Set Host parameter if it exists
-        if($HostHeader -ne $null)
+        if ($HostHeader -ne $null)
         {
-            $bindingParams.Add('-HostHeader', $HostHeader)
+            $BindingParams.Add('HostHeader', $HostHeader)
             $UseHostHeader = $true
         }
 
-        if(-not [string]::IsNullOrWhiteSpace($SSLFlags))
+        if ([Environment]::OSVersion.Version -ge '6.2' -and -not [String]::IsNullOrWhiteSpace($SslFlags))
```

Comment:
According to the [official IIS site](https://www.iis.net/configreference/system.applicationhost/sites/site/bindings/binding) the **sslFlags** attribute was added in IIS 8.0 that is packaged with Windows Server 2012 and Windows 8.
That's why we need to ensure the OS version number is greater or equal to 6.2 (see [Operating System Version](https://msdn.microsoft.com/en-us/library/windows/desktop/ms724832%28v=vs.85%29.aspx) for reference).